content_scripts/link_hints.coffee

```diff
@@ -625,6 +625,9 @@ LocalHints =
           if element.clientHeight < element.scrollHeight and Scroller.isScrollableElement element
             reason = "Scroll."
 
+      when "pre"
+        isClickable ||= !!(element.clientWidth - element.scrollWidth) || !!(element.clientHeight - element.scrollHeight)
+
```

Comment:
@MuhanZou The code immediately above is doing the same thing, except for the y-axis only.  We should do this once only.  Probably we'd be better off generalising the code above rather than adding a new case.